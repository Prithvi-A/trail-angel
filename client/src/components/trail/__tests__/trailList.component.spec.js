import React from 'react';
import renderer from 'react-test-renderer';

import TrailList from '../trailList.component';


describe('Trail List component', () => {
  it('renders correctly', () => {

    const tree = renderer.create(
      <TrailList />
    ).toJSON();

    expect(tree).toMatchSnapshot();

  });
});
