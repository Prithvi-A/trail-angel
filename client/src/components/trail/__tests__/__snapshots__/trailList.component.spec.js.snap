exports[`Trail List component renders correctly 1`] = `
<View>
  <View
    style={Object {}}>
    <Image
      source={1}
      style={
        Object {
          "marginTop": 65,
        }
      } />
    <ScrollView
      dataSource={
        ListViewDataSource {
          "items": "unknown",
        }
      }
      enableEmptySections={true}
      initialListSize={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={1000}
      onKeyboardDidHide={undefined}
      onKeyboardDidShow={undefined}
      onKeyboardWillHide={undefined}
      onKeyboardWillShow={undefined}
      onLayout={[Function]}
      onScroll={[Function]}
      pageSize={1}
      removeClippedSubviews={true}
      renderRow={[Function]}
      renderSeparator={[Function]}
      scrollEventThrottle={50}
      scrollRenderAheadDistance={1000}
      stickyHeaderIndices={Array []}
      style={Object {}} />
  </View>
</View>
`;
